cypress/integration/sidebar.spec.js

```diff
@@ -0,0 +1,26 @@
+const autoRecord = require('cypress-autorecord');
+
+describe('The Sidebar', () => {
+  autoRecord();
+
+  before(() => {
```

Comment:
@rafayet-monon How about using `beforeEach` here and update the test `when the sidebar is already open` to be independent of the previous case as in compass mentioned about [each test must run in isolation](https://nimblehq.co/compass/development/testing/#general-best-practices)? 🙌🏼